tools/applicationinsights-web-snippet/src/snippet.ts

```diff
@@ -194,12 +207,29 @@ declare var cfg:ISnippetConfig;
                 });
                 // let message = "Load Version 2 SDK instead to support IE"; // where to report this error?
             }
+
+            if (cfg.cr){
+                for (var i = 0; i < domains.length; i++){
+                    if (targetSrc.indexOf(domains[i]) > 0){
+                        domainRetryIndex = i;
+                        break;
+                    }
+                }
+            }
+
             const _handleError = (evt?: any) => {
-                loadFailed = true;
                 appInsights.queue = []; // Clear the queue
                 if (!handled) {
-                    handled = true;
-                    _reportFailure(targetSrc);
+                    // start retry
+                    if (domainRetryIndex >= 0 && domainRetryCount < domains.length){
+                        let nextIdx = (domainRetryIndex + domainRetryCount + 1) % domains.length;
+                        _createScript("https://" + domains[nextIdx] + "/scripts/b/ai.2.min.js");
```

Comment:
But this bit should not always default to `/scripts/b/ai.2.min.js` it should always be whatever they supplied as it could also be
- version 3
- one of the other formats (like `ai.2.gbl.min.js`)

Reply:
So maybe a regex like the IE fallback

```
_createScript(targetSrc.replace(/^(.*\/\/)([\w\.]*)(\/.*)$/, function (_all, http, domain, qs) {
    return http + domains[nextIdx] + qs;
});
```